bundles/org.openhab.binding.rotel/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,178 @@
+			<channel id="recordSource" typeId="recordSource" />
+			<channel id="dsp" typeId="dsp" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+			<channel id="line1" typeId="frontPanelLine" />
+			<channel id="line2" typeId="frontPanelLine" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="mainZoneType3">
+		<label>Main Zone</label>
+		<description>The controls of the main zone</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="recordSource" typeId="recordSource" />
+			<channel id="dsp" typeId="dsp" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+			<channel id="bass" typeId="bass" />
+			<channel id="treble" typeId="treble" />
+			<channel id="line1" typeId="frontPanelLine" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone2type1">
+		<label>Zone 2</label>
+		<description>The controls of the zone 2</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone2type2">
+		<label>Zone 2</label>
+		<description>The controls of the zone 2</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone3">
+		<label>Zone 3</label>
+		<description>The controls of the zone 3</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone4">
+		<label>Zone 4</label>
+		<description>The controls of the zone 4</description>
+		<channels>
+			<channel id="power" typeId="power" />
+			<channel id="source" typeId="source" />
+			<channel id="volume" typeId="volume" />
+			<channel id="mute" typeId="mute" />
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power ON/OFF the equipment or the zone</description>
+	</channel-type>
+
+	<channel-type id="source">
+		<item-type>String</item-type>
+		<label>Source Input</label>
+		<description>Select the source input</description>
+	</channel-type>
+
+	<channel-type id="recordSource" advanced="true">
+		<item-type>String</item-type>
+		<label>Record Source</label>
+		<description>Select the source to be recorded</description>
+	</channel-type>
+
+	<channel-type id="dsp">
+		<item-type>String</item-type>
+		<label>DSP Mode</label>
+		<description>Select the DSP mode</description>
+	</channel-type>
+
+	<channel-type id="volume">
```

Comment:
You should not define your own volume channel type and instead use the [system channel](https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types) `system.volume` for it.
This also means that volume channels have to be Dimmers, thus you can remove all the min/max information about them.

Reply:
I have to remember why I was forced to add the VETO autoUpdatePolicy property.
But yes, I will move to Dimmer channel and if possible to the system.volume channel, at least for almost all Rotel devices, all those having a direct volume set command.
I could keep this specific channel for the rare models (including mime) that have only UP and DOWN commands for volume and so have to be linked to a setpoint channel.
It requires to consider the translation of the value to 0-100 range too.

Reply:
Done.
I kept special channels only for 3 Rotel models for which a dimmer is not appropriate. All others models are now using the system.volume channel.
Waiting for a feedback to be sure that iot works as expected.